rpc/block_create.c

```diff
@@ -1047,7 +1079,28 @@ block_create_cli_1_svc_st(blockCreateCli *blk, struct svc_req *rqstp)
   GB_STRCPYSTATIC(cobj.gbid, gbid);
   GB_STRDUP(cobj.block_hosts,  blk->block_hosts);
 
-  if (blk->blk_size) { // Create V3
+  if (!resultCaps[GB_CREATE_TCMUR_TIMEOUT_CAP]) { // Create V4
+    unsigned int len;
+    struct gbCreate3 *gbCreate3;
+
+    if (!blk->io_timeout)
+	    blk->io_timeout = 43;
```

Comment:
can this be a macro ? may be like GB_IO_TIMEOUT ? also fix the indent to 4 spaces.

Reply:
Done.